handlers/accesscontrol/encryption.go

```diff
@@ -25,9 +26,17 @@ func (ec *EncryptionHandlersCollection) PublicKeyHandler() httprouter.Handle {
 		w.Header().Set("Content-Type", "application/json")
 		w.WriteHeader(http.StatusOK)
 
+		spkiPublicKey, err := crypto.ConvertToSpki(ec.publicKey)
```

Comment:
Can you call this on the constructor instead? The inputs are constant so we don't need to be recomputing this conversion on every request.

Reply:
Moved into the handler constructor